packages/edit-site/src/components/navigation-sidebar/navigation-panel/menus/templates.js

```diff
@@ -59,25 +64,18 @@ export default function TemplatesMenu( { onActivateItem } ) {
 				title={ __( 'Posts' ) }
 			/>
 
-			<TemplateNavigationItems
-				templates={ generalTemplates }
-				onActivateItem={ onActivateItem }
-			/>
+			{ map( generalTemplates, ( template ) => (
+				<TemplateNavigationItem
+					item={ template }
+					key={ `wp_template-${ template.id }` }
+				/>
+			) ) }
 
-			<TemplatePostsMenu
-				templates={ templates }
-				onActivateItem={ onActivateItem }
-			/>
+			<TemplatePostsMenu templates={ templates } />
```

Comment:
A total nitpick on something not introduced here.  TemplatesPagesMenu and TemplatesAllMenu names are plural (for templates) while this one is both defined and imported as singular.